src/core/thread/mesh_forwarder.hpp

```diff
@@ -140,7 +141,7 @@ class MeshForwarder : public InstanceLocator
      * @param[in]  aScanChannels  A bit vector indicating which channels to scan.
```

Comment:
nit: changing the comment about the paramter.
Also maybe adding a comment that if `aScanChannels` is empty, then all channels are used instead.

Reply:
Changed.